### PR TITLE
Screenshots changes language in the application

### LINKED ISSUE #84: Create screenshots for all the views in all languages and publish-it for review
For easy review it would be create to create UI test that will screenshot all views in all languages and publishes the index.html on GitHub Pages for a review (and send a slack message to #ua-projekty-preklady-do-vsech-jazyku)

### LINKED ISSUE #172: Deploy GHA často nedokončí screenshots
GitHub Action Deploy dost často nedokončí screenshot pipeline a vytimeoutuje.
Teorie:
- Buildí se pokaždé appka když se přepne jazyk
- Resetuje se simulátor

### PR DESCRIPTION
**Pull request by kubito**
Changes application language when using fastlane screenshots.

### CODE REVIEW

File: Movapp.xcodeproj/xcshareddata/xcschemes/MovappScreenshots.xcscheme [hunk 1]
```diff
@@ -37,17 +37,14 @@
                BlueprintName = "MovappUITests"
                ReferencedContainer = "container:Movapp.xcodeproj">
             </BuildableReference>
-            <SkippedTests>
+            <SelectedTests>
                <Test
-                  Identifier = "MovappUITests">
+                  Identifier = "FastlaneSnapshot/testExample()">
```
Comment (pionl) [inline Movapp.xcodeproj/xcshareddata/xcschemes/MovappScreenshots.xcscheme:42]: is this correct?

**Comment by pionl** [comment]
Well I love the idea, but not sure how fastlane works - this will run fastlane screenshot, build it only once and change language programatically on running simulator?

By the code I have no idea what it does 🗡️

File: Movapp.xcodeproj/xcshareddata/xcschemes/MovappScreenshots.xcscheme (hunk 1 above)
Comment (kubito) [reply Movapp.xcodeproj/xcshareddata/xcschemes/MovappScreenshots.xcscheme:42]: This means, that only this one particular test is included in the MovappScreenshots scheme so if we will add a new test we need to add it manually for using it for screenshots.

**Comment by kubito** [comment]
> Well I love the idea, but not sure how fastlane works - this will run fastlane screenshot, build it only once and change language programatically on running simulator?
> 
> By the code I have no idea what it does 🗡️


Fastlane capture_screenshots works in a way where it builds the application first and then runs the application on a simulator with a different configuration (based on Snapfile). Run the same test again and again just by injecting different parameters into the simulator and runtime. So you need to check and parse those parameters on the startup of the application to modify the settings.

File: Fastlane/Snapfile [hunk 2]
```diff
@@ -2,9 +2,10 @@
...
-  "iPhone 14 Pro Max",
-  "iPhone 8 Plus",
-  "iPad Pro (12.9-inch) (6th generation)"
+  "iPhone 14 Pro Max", # Optional for Apple review
+  "iPhone 13 Pro Max", # Mandatory for Apple review
+  "iPhone 8 Plus", # Mandatory for Apple review
+  "iPad Pro (12.9-inch) (6th generation)" # Mandatory for Apple review
 ])
 # TODO we need to generate screenshot for iPad but Github is missing this
 #"iPad Pro (12.9-inch) (2nd generation)"
```
Comment (pionl) [inline Fastlane/Snapfile:11]: @kubito nenapadá tě jak toto vyřešit? Pro review je to nutné. Mě napadlo přes command line zaregistrovat / naiinstalovat ten simulátor 🤔

**Review by pionl — APPROVED** [review]

**Pull request merged**